src/Service/BuyService.php

```diff
@@ -35,21 +35,21 @@ public function __construct(
         $this->configuredExchange = $configuredExchange;
     }
 
-    public function buy(int $amount, string $tag = null): CompletedBuyOrder
+    public function buy(int $amount, string $asset, string $tag = null): CompletedBuyOrder
     {
         $logContext = [
             'exchange' => $this->configuredExchange,
             'amount' => $amount,
+            'asset' => $asset,
             'tag' => $tag,
         ];
 
-        $this->logger->info('performing buy for {amount}', $logContext);
+        $this->logger->info('performing buy for {amount} of token {token}', $logContext);
```

Comment:
Hier wordt {token} gebruikt maar in je context is `asset` toegevoegd.

Reply:
Inderdaad, een typo